### ISSUE TITLE
MiniAudicle build error on Fedora 38

### ISSUE THREAD

**Issue by ycollet** [ccrma/miniAudicle#76]
I am building miniAudicle 1.5.2.1 on Fedora 38 and I met a build error:
```
make[1]: Entering directory '/home/collette/rpmbuild/BUILD/miniAudicle/src'
g++ -c -pipe -I/home/collette/rpmbuild/BUILD/miniAudicle/src/qt -D__LINUX_ALSA__ -D__CK_SNDFILE_NATIVE__ -Ichuck/src/core -Ichuck/src/host -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -O3 -w -D_REENTRANT -D__MA_COLOR_CONSOLE__ -DHAVE_CONFIG_H -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I. -I/usr/include/qt6 -I/usr/include/qt6/QtWidgets -I/usr/include/qt6/QtGui -I/usr/include/qt6/QtNetwork -I/usr/include/qt6/QtCore -Iqt-build -Iqt-build -I/usr/lib64/qt6/mkspecs/linux-g++ -o qt-build/mAConsoleMonitor.o qt/mAConsoleMonitor.cpp
qt/mAConsoleMonitor.cpp: In constructor 'mAConsoleMonitor::mAConsoleMonitor(QWidget*, miniAudicle*)':
qt/mAConsoleMonitor.cpp:89:17: error: 'CK_LOG_SEVERE' was not declared in this scope; did you mean 'CK_LOG_CORE'?
   89 |         EM_log( CK_LOG_SEVERE, "(console monitor): pipe error, disabling console monitor" );
      |                 ^~~~~~~~~~~~~
      |                 CK_LOG_CORE
make[1]: *** [makefile.qt-alsa:2240: qt-build/mAConsoleMonitor.o] Error 1
```

gcc version 13.2.1.

**Comment by gewang** [comment]
thanks for reporting this issue; a fix was pushed to `main`, and the `chuck-1.5.2.1` tag has been updated to include this fix. please let us know if that fixes the build!

**Comment by ycollet** [comment]
Build fine now. Thanks for the fix !
